lib/generators/alchemy/install/files/all.js

```diff
@@ -9,3 +9,5 @@
 // about supported directives.
 //
 //= require alchemy/admin
+//= require alchemy_i18n/de
+//= require select2_locale_de
```

Comment:
This should not be in this generator. The `alchemy_i18n` gem has its own generator.

```suggestion
```